src/Materials/Node/Blocks/PBR/subSurfaceBlock.ts

```diff
@@ -227,6 +229,8 @@ export class SubSurfaceBlock extends NodeMaterialBlock {
                 #ifdef SS_LINKREFRACTIONTOTRANSPARENCY
                     alpha = subSurfaceOut.alpha;
                 #endif
+            #elif defined(SS_TRANSLUCENCY)
+                surfaceAlbedo = subSurfaceOut.surfaceAlbedo;

```

Comment:
You don't need to add those two lines because the TRANSLUCENCY path in the subsurface block does not update the albedo color (whereas the REFRACTION path does).

Reply:
Oops. That was left over from some of the scattering approximation stuff. Thanks!